perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -575,53 +535,23 @@ public void moveGroup(PerunSession sess, Group destinationGroup, Group movingGro
 				}
 			}
 
-			//After that, we have to remove indirect members from old position
-			List<Member> members = getGroupMembers(sess, movingGroup);
+			processRelationsWhileMovingGroup(sess, destinationGroup, movingGroup);
```

Comment:
Since I can't comment on unchanged lines...at line 531 there is a comment ``// check if under destination group is group with same short name as Moving group short name`` which is a little confusing, since this branch is resolving moving to the top-level.